src/adapters/bidderFactory.js

```diff
@@ -107,6 +107,8 @@ import { logWarn, logError, parseQueryStringParameters, delayExecution } from 's
  * @property {string} url The URL which makes the sync happen.
  */
 
+const BID_RESPONSE_KEYS = ['requestId', 'bidderCode', 'cpm', 'width', 'height', 'ad', 'ttl', 'creativeId', 'netRevenue', 'currency'];
```

Comment:
Per recent discussion, it looks like `bidderCode` should not be a required param.